### PR TITLE
Separate unit from performance tests

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: CommonCPPCTest.cmake [hunk 1]
```diff
@@ -52,7 +67,7 @@ foreach(FILE ${TEST_FILES})
   # Test link libraries
   target_link_libraries(${PROJECT_NAME}_${NAME}
                     ${${NAME}_LINK_LIBRARIES} ${TEST_LIBRARIES})
-  # Per target test command customisation with
+   # Per target test command customisation with
```
Comment (tribal-tec) [inline CommonCPPCTest.cmake]: space?

**Comment by tribal-tec** [comment]
+1

File: CommonCPPCTest.cmake (hunk 1 above)
Comment (eile) [inline CommonCPPCTest.cmake]: fixed

**Comment by eile** [comment]
Merged with direct push.

**Pull request closed without merging**